### DISCUSSION TITLE
twinBASIC exposure welcome on Discord

### DISCUSSION THREAD

**Discussion by retailcoder** [twinbasic/twinbasic#1484] [General]
One of my Twitter followers has set up a VB-focused discord server for everything that relates to Visual Basic, and they were very happy to see me join and created a dedicated discussion page for Rubberduck - twinBASIC is welcome too, and would receive similar treatment. Here's a link if you're interested: https://discord.gg/DmtTg3Mj

**Comment by fafalone** [comment]
There's a twinBASIC Discord that's quite active... more active than this GitHub for a while now..

https://github.com/twinbasic/twinbasic/discussions/629

**Reply by retailcoder** [reply]
Ah, been out of the loop and watching from email notifications for a while... I must have added it back when it was first posted - I had apparently even already joined it, but never really got onto Discord until now.. guess it only took Twitter collapsing 😂
